packages/office-ui-fabric-react/src/components/Nav/examples/Nav.Basic.Example.tsx

```diff
@@ -83,4 +84,7 @@ function _onLinkClick(ev: React.MouseEvent<HTMLElement>, item?: INavLink) {
   if (item && item.name === 'News') {
     alert('News link clicked');
   }
+  if (_nav && _nav.current) {
+    _nav.current.focus(true);
```

Comment:
minor nit here; can you please remove this from the example? It's nice to test the functionality but makes for a weird experience for the customer when they're looking at the example. (They would click on News, and suddenly focus would move unnaturally.

Reply:
done